AppRentCar/forms.py

```diff
@@ -0,0 +1,12 @@
+from django import forms
+
+from .models import Car
+
+
+class CarForm(forms.ModelForm):
+    class Meta:
+        model = Car
+        fields = '__all__'
```

Comment:
lepiej jest wypisać wszystkie pola, `explicit is better than implicit`, komentarz tyczy się wszystkich użyć takich w kodzie